protoc_plugin/test/duration_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:fixnum/fixnum.dart';
+import 'package:protoc_plugin/test/protos/google/protobuf/duration.pb.dart'
```

Comment:
Imports are different in the g3 and OS versions. This should be something like: `import '../out/protos/google/protobuf/duration.pb.dart';`